src/app/shared/components/template/components/toggle-bar/toggle-bar.ts

```diff
@@ -66,10 +69,12 @@ export class TmplToggleBarComponent
       "show_tick_and_cross",
       true
     );
+    this.params.showIcons = getBooleanParamFromTemplateRow(this._row, "show_icons", true);
     this.params.style = getStringParamFromTemplateRow(this._row, "style", "");
     this.params.variant = getStringParamFromTemplateRow(this._row, "variant", "")
       .split(",")
       .join(" ") as IToggleParams["variant"];
+    this.platformVariant = this.params.variant.split(" ").includes("android") ? "md" : "ios";
```

Comment:
nit (non-blocking): Maybe worth adding an explanatory comment above this line, something like:
```ts
// The ion-toggle component uses "md" ("material design") and "ios" to refer to visual styles of the component corresponding to the respective platforms.
// see docs here: https://ionicframework.com/docs/api/toggle
```

Reply:
(non-blocking): I can see that the _default_ would be `ios` (unless android specified) - which would include behaviour on web. I think given more of our user-base is android makes sense for that to be the default on web also - but not a huge issue either way (the ios style is still nice)